lvsfunc.py

```diff
@@ -1,20 +1,47 @@
 """
-    Various functions I make use of often and other people might be able to use too. Suggestions and fixes are always appreciated!
+    Various functions I make use of often and other people might be able to use too.
+    Suggestions and fixes are always appreciated!
 """
 
-import vapoursynth as vs
-import kagefunc as kgf # https://github.com/Irrational-Encoding-Wizardry/kagefunc
-import vsutil # https://github.com/Irrational-Encoding-Wizardry/vsutil
+from functools import partial
+
+import fvsfunc as fvf  # https://github.com/Irrational-Encoding-Wizardry/fvsfunc
+import havsfunc as hvf  # https://github.com/HomeOfVapourSynthEvolution/havsfunc
+import kagefunc as kgf  # https://github.com/Irrational-Encoding-Wizardry/kagefunc
+import mvsfunc as mvf  # https://github.com/HomeOfVapourSynthEvolution/mvsfunc
+from vsTAAmbk import TAAmbk  # https://github.com/HomeOfVapourSynthEvolution/vsTAAmbk
+from vsutil import *  # https://github.com/Irrational-Encoding-Wizardry/vsutil
+
+from nnedi3_rpow2 import \
+    nnedi3_rpow2  # https://github.com/darealshinji/vapoursynth-plugins/blob/master/scripts/nnedi3_rpow2.py
+
 core = vs.core
+# optional requires: (http://www.vapoursynth.com/doc/pluginlist.html)
+#   waifu2x-caffe
+#   L-SMASH Source
+#   d2vsource
+#   FFMS2
 
-# TO-DO: Write function that only masks px of a certain color/threshold of colors
+# TODO: Write function that only masks px of a certain color/threshold of colors
 
 
-def compare(clip_a: vs.VideoNode, clip_b: vs.VideoNode, frames: int, mark=False, mark_a=' Clip A ', mark_b=' Clip B ', fontsize=57):
+def compare(clip_a: vs.VideoNode, clip_b: vs.VideoNode, *frames: int, mark: bool = False, mark_a: str = ' Clip A ',
+            mark_b: str = ' Clip B ',
+            fontsize: int = 57) -> vs.VideoNode:
     """
     Allows for the same frames from two different clips to be compared by putting them next to each other in a list.
-
+    
     Shorthand for this function is "comp".
+
+    :param clip_a: vs.VideoNode: 
```

Comment:
I don’t think these docstrings add any kind of value. Types and default values are already in the function header. This just clutters the file and reminds me of horrible java enterprise conventions